### PR TITLE
feat: linux 6.1.1, mpc 1.3.1

### PR DESCRIPTION
**Pull request by smira**
Routing updates, but jumping to the next LTS.

Signed-off-by: Andrey Smirnov <andrey.smirnov@talos-systems.com>

### CODE REVIEW

File: Pkgfile [hunk 1]
```diff
@@ -27,14 +27,14 @@ vars:
...
-  mpc_sha512: 9c18b24f7542dc1dc5e10cf58fd242e73d79a9dc3619c3f08d52aed75ad0e7d9e2ba2c46857717c8b921b084af2efc8c0d2d7173081af764b81c24a8971ddd9a
+  mpc_version: 1.3.1
+  mpc_sha256: ab642492f5cf882b74aa0cb730cd410a81edcdbec895183ce930e706c1c759b8
+  mpc_sha512: 4bab4ef6076f8c5dfdc99d810b51108ced61ea2942ba0c1c932d624360a5473df20d32b300fc76f2ba4aa2a97e1f275c9fd494a1ba9f07c4cb2ad7ceaeb1ae97
 
   # renovate: datasource=git-tags extractVersion=^v(?<version>.*)$ depName=git://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git
-  linux_version: 5.15.82
-  linux_sha256: fceef6bb79bac494663ccde34453521fc616cd94272fd30564752b3742381b65
-  linux_sha512: b0deb17077d9254e9a6eef853cdbcb7cbdde74cafef214d25961929d02a42fd61d306e3358b17a145999a0df3565c985de6149bd078330c63508ce8ce6fb4938
+  linux_version: 6.1.1
```
Comment (frezbo) [inline Pkgfile:35]: still in mainline it seems? :thinking:

**Review by frezbo — APPROVED** [review]

File: Pkgfile (hunk 1 above)
Comment (smira) [reply Pkgfile:35]: let me double check, but as it has 'stable' release now, it should be

Comment (smira) [reply Pkgfile:35]: yep, it's there https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/tag/?h=v6.1.1

Comment (frezbo) [reply Pkgfile:35]: i meant it's still not tagged as LTS :sweat_smile:

Comment (smira) [reply Pkgfile:35]: the decision haven't been made, in the worst case it will be 6.2, so still in time for Talos 1.4

**Pull request merged**